man/systemd.service.xml

```diff
@@ -1088,6 +1088,279 @@ ExecStart=/bin/echo $ONE $TWO $THREE</programlisting>
+
+    <para>The following are the basic service exit codes defined by the
+      <ulink
+        url="https://refspecs.linuxbase.org/LSB_5.0.0/LSB-Core-generic/LSB-Core-generic/iniscrptact.html">LSB specification
+      </ulink>.
+    </para>
+
+    <table>
+      <title>LSB service exit codes</title>
+      <tgroup cols='3'>
+        <thead>
+          <row>
+            <entry>Exit Code</entry>
+            <entry>Symbolic Name</entry>
+            <entry>Description</entry>
+          </row>
+        </thead>
+        <tbody>
+          <row>
+            <entry>0</entry>
+            <entry>EXIT_SUCCESS</entry>
+            <entry>Generic success code. Defined by the C library implementation.</entry>
+          </row>
+          <row>
+            <entry>1</entry>
+            <entry>EXIT_FAILURE</entry>
+            <entry>Generic failure or unspecified error. Defined by the C library implementation.</entry>
+          </row>
+          <row>
+            <entry>2</entry>
+            <entry>EXIT_INVALIDARGUMENT</entry>
+            <entry>Invalid or excess arguments.</entry>
+          </row>
+          <row>
+            <entry>3</entry>
+            <entry>EXIT_NOTIMPLEMENTED</entry>
+            <entry>Unimplemented feature.</entry>
+          </row>
+          <row>
+            <entry>4</entry>
+            <entry>EXIT_NOPERMISSION</entry>
+            <entry>The user has insufficient privileges.</entry>
+          </row>
+          <row>
+            <entry>5</entry>
+            <entry>EXIT_NOTINSTALLED</entry>
+            <entry>The program is not installed.</entry>
+          </row>
+          <row>
+            <entry>6</entry>
+            <entry>EXIT_NOTCONFIGURED</entry>
+            <entry>The program is not configured.</entry>
+          </row>
+          <row>
+            <entry>7</entry>
+            <entry>EXIT_NOTRUNNING</entry>
+            <entry>The program is not running.</entry>
+          </row>
+        </tbody>
+      </tgroup>
+    </table>
+
+    <para>
+      The LSB suggests that error codes over 200 (included) are reserved. These are the return codes specified by systemd.
+    </para>
+    <table>
+      <title>systemd-specific exit codes</title>
+      <tgroup cols='3'>
+        <thead>
+          <row>
+            <entry>Exit Code</entry>
+            <entry>Symbolic Name</entry>
+            <entry>Description</entry>
+          </row>
+        </thead>
+        <tbody>
+          <row>
+            <entry>200</entry>
+            <entry>EXIT_CHDIR</entry>
+            <entry>The chdir(2) call failed.</entry>
+          </row>
+          <row>
+            <entry>201</entry>
+            <entry>EXIT_NICE</entry>
+            <entry>Failed to set the scheduling priority of a process.</entry>
+          </row>
+          <row>
+            <entry>202</entry>
+            <entry>EXIT_FDS</entry>
+            <entry>Closing up of file descriptors (TODO: Which ones??) failed.</entry>
```

Comment:
We close all fds except the ones that we specifically want to pass to the process. I'd just say "Closing of unwanted file descriptors failed."